codewof/programming/content/en/long-count/solution.py

```diff
@@ -0,0 +1,5 @@
+def long_count(list):
+    length = 0
+    for _ in list:
```

Comment:
I think using an `_` here would be confusing as users are unlikely to have seen style like this before. I'd suggest changing it to `item`